src/bridge/cockpitpolkithelper.c

```diff
@@ -144,7 +145,8 @@ main (int argc, char *argv[])
 
       if (ferror (stdout))
         {
-          warnx ("couldn't write to stdout");
+          if (errno != EPIPE)
```

Comment:
I think this is actually OK and errno still reflects the fflush call since neither free nor ferror set errno.  For extra clarity, we might explicitly store it in a local variable after fflush, though.